roles/first_deploy/tasks/main.yml

```diff
@@ -97,6 +118,11 @@
   args:
     chdir: /home/{{ ansible_ssh_user }}/{{ project_name }}
 
+- name: install sidekiq systemd user service for deploy user
+  shell: BRANCH={{ branch | default('master') }} cap localhost sidekiq:install
+  args:
+    chdir: /home/{{ ansible_ssh_user }}/{{ project_name }}
+
```

Comment:
It looks like this duplicates lines 87-125.  Can we add a comment if both are really required; otherwise, I'm afraid someone would think they're unnecessary duplication and delete one.